modules/odc-creating-apiserversource.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * applications/application_life_cycle_management/odc-knative-event-sources.adoc
+
+[id="odc-creating-apiserversource_{context}"]
+= Creating an ApiServerSource event source using the Developer perspective
+
+.Procedure
+
+. Navigate to the **Add** page.
++
+image::event-sources-odc.png[Add an Event Source in the Developer perspective]
+. Select **Event Sources** -> **ApiServerSource**.
```

Comment:
s/**Event Sources** /**Event Source**:  In the Add page it is **Event Source** which then ironically takes you to the **Event Sources** page. I will let the Devs know that it should be made consistent.

Reply:
Alternatively, first step could be: Navigate to the **Add** page and select **Event Source**.
and second could be: In the **Event Sources** page, select **ApiServerSource** in the **Type** section.